_translator-files/po/sv-SE/1-index.po

```diff
@@ -81,33 +98,35 @@ msgid ""
 "Check out the [documentation](wiki/Getting-Started) and consider the [troubleshooting section](wiki/Client-Troubleshooting)!\n"
 "You can also ask on the [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 msgstr ""
+"Kolla in [dokumentationen](wiki/Getting-Started) och överväg att läsa avsnittet om [troubleshooting section](wiki/Client-Troubleshooting)! \n"
+"Du kan också ställa frågor på [forums](https://github.com/jamulussoftware/jamulus/discussions).\n"
 
 #. type: Content of: <div><div><div><h2>
 #: ../wiki/en/misc/1-index.md:53
 #, no-wrap
 msgid "Want to get involved?"
-msgstr ""
+msgstr "Vill du engagera dig?"
 
 #. type: Content of: <div><div><div><p><div>
 #: ../wiki/en/misc/1-index.md:56
 #, no-wrap
 msgid "Ideas? Found a bug? Want to contribute some code or help [translating](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md) Jamulus into your language? Since Jamulus is [free and open source software](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensed under the [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) you can help us!"
-msgstr ""
+msgstr "Har du idéer? Hittat en bugg? Vill du bidra med kod eller hjälpa till att [translating](https://github.com/jamulussoftware/jamulus/blob/main/docs/TRANSLATING.md) Jamulus till ditt språk? Eftersom Jamulus är [fri och öppen källkodsprogramvara](https://www.gnu.org/philosophy/free-sw.en.html) (FOSS) licensierad under [GPL](https://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html) kan du hjälpa oss!"
 
 #. type: Content of: <div><div><div><p><div><br><br>
 #: ../wiki/en/misc/1-index.md:58
 #, no-wrap
 msgid "Take a look at our [contribution guidelines](wiki/Contribution) to find out how. Everybody is welcome!"
-msgstr ""
+msgstr "Ta en titt på våra [contribution guidelines](wiki/Contribution) för att ta reda på hur du kan bidra. Alla är välkomna!"
```

Comment:
Content in square brackets needs translating.

Reply:
`id="bidra-till-jamulusprojektet"`